tests/unittest_nnstreamer-edge.cc

```diff
@@ -3743,7 +3743,7 @@ _check_mqtt_broker ()
 /**
  * @brief Connect to the local host using the information received from mqtt.
  */
-TEST(edgeMqtt, connectLocal)
+TEST(edgeMqttHybrid, connectLocal)
```

Comment:
How about removing all mqtt tests in this file?
IMO, negative cases are duplicated.

Reply:
I moved mqtt realted tests from `unittest_nnstreamer-edge.cc` to `unittest_nnstreamer-edge-mqtt.cc` and remove duplicated nagative tests. Thanks.